OptimizelySDKCore/OptimizelySDKCoreTests/OptimizelyTest.m

```diff
@@ -1182,6 +1193,123 @@ -(void)testGetEnabledFeaturesWithSomeFeaturesEnabledForUser {
     XCTAssertEqualObjects(features, enabledFeatures);
 }
 
+#pragma mark - TypeAudiences Tests
```

Comment:
Curious. Can you explain the set of tests that you have here? I see most are testing for the "happy path" i.e. everything matches and works correctly. Is that correct?

Reply:
We have tests both for typed audience inclusion and exclusion in all APIs here. Am I missing anything?